gradle/libs.versions.toml

```diff
@@ -1,20 +1,19 @@
 [versions]
 kotlin = "1.9.0"
-spring-cloud-gcp = "4.5.1"
 testcontainers = "1.18.3"
 
 [libraries]
 plugin-kotlin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
 plugin-kotlin-allopen = { module = "org.jetbrains.kotlin:kotlin-allopen", version.ref = "kotlin" }
 plugin-dokka = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version = "1.8.20" }
 plugin-spring-boot = { module = "org.springframework.boot:spring-boot-gradle-plugin", version = "3.1.1" }
-plugin-spring-dependency-management = { module = "io.spring.gradle:dependency-management-plugin", version = "1.1.2" }
+plugin-spring-dependency-management = { module = "io.spring.gradle:dependency-management-plugin", version = "1.1.0" }
```

Comment:
Why are we downgrading this plugin? Does this help with something that isn't documented?

Reply:
Sorry, it's my carelessness.